packages/fx-core/templates/plugins/resource/function/bicep/function.template.bicep

```diff
@@ -0,0 +1,120 @@
+param functionServerfarmsName string
+param functionAppName string
+@minLength(3)
+@maxLength(24)
+@description('Name of Storage Accounts for function backend.')
+param functionStorageName string
+param aadClientId string
+@secure()
+param aadClientSecret string
+param m365TenantId string
+param applicationIdUri string
+
+{{#contains 'fx-resource-frontend-hosting' Plugins}}
+param frontendHostingStorageEndpoint string
+{{/contains}}
+{{#contains 'fx-resource-azure-sql' Plugins}}
+param sqlDatabaseName string
+param sqlEndpoint string
+{{/contains}}
+{{#contains 'fx-resource-identity' Plugins}}
+param identityId string
+{{/contains}}
+
+var oauthAuthorityHost = environment().authentication.loginEndpoint
```

Comment:
`oauthAuthorityHost` has been moved to be an input parameter which is passed in by aad plugin.